tools/phenomenal/isatools/isa_create_metabo/isa_create_metabo.xml

```diff
@@ -7,7 +7,7 @@
 
     <requirements>
```

Comment:
I think that having these requirements is what confuses the container selector when using dynamic destinations...